packages/forms/src/form_builder.ts

```diff
@@ -213,6 +217,39 @@ export class FormBuilder {
     return new FormGroup(reducedControls, {asyncValidators, updateOn, validators}) as any;
   }
 
+  /**
+   * @description
+   * Construct a new `FormRecord` instance. Accepts a single generic argument, which is an object
+   * containing all the keys and corresponding inner control types.
+   *
+   * @param controls A collection of child controls. The key for each child is the name
+   * under which it is registered.
+   *
+   * @param options Configuration options object for the `FormRecord`. The object should have the
+   * `AbstractControlOptions` type and might contain the following fields:
+   * * `validators`: A synchronous validator function, or an array of validator functions.
+   * * `asyncValidators`: A single async validator or array of async validator functions.
+   * * `updateOn`: The event upon which the control should be updated (options: 'change' | 'blur'
+   * | submit').
+   */
+  record<T>(controls: {[key: string]: T}, options: AbstractControlOptions|null = null):
+      FormRecord<ɵElement<T, null>> {
+    const reducedControls = this._reduceControls(controls);
+
+    let validators: ValidatorFn|ValidatorFn[]|null = null;
+    let asyncValidators: AsyncValidatorFn|AsyncValidatorFn[]|null = null;
+    let updateOn: FormHooks|undefined = undefined;
+
+    if (options !== null) {
+      validators = options.validators != null ? options.validators : null;
+      asyncValidators = options.asyncValidators != null ? options.asyncValidators : null;
+      updateOn = options.updateOn != null ? options.updateOn : undefined;
+    }
+
+    // Cast to `any` because the inferred types are not as specific as Element.
+    return new FormRecord(reducedControls, {asyncValidators, updateOn, validators}) as any;
```

Comment:
I'm wondering if we can just pass the `options` object to the `FormRecord`?
```suggestion
    // Cast to `any` because the inferred types are not as specific as Element.
    return new FormRecord(reducedControls, options) as any;
```

Reply:
Right 👍 I updated the PR, if you want to take another look.

Reply:
@cexbrayat if you are interested and have a chance to contribute, we can do a similar code optimization (in a separate PR) in the `group` function of the `FormBuilder` (similar to the code that we have in the `control` function).